tko_web_sessions_management/main.py

```diff
@@ -164,11 +171,14 @@ def web_login(self, redirect=None, **kw):
                 request.context)
             _logger.error(unsuccessful_message)
             request.uid = old_uid
-            values['error'] = 'Login failed due to one of the following reasons:'
-            values['reason1'] = '- Wrong login/password'
-            values['reason2'] = '- User not allowed to have multiple logins'
-            values[
-                'reason3'] = '- User not allowed to login at this specific time or day'
+            values['error'] = _('Login failed due to one of the following reasons:')
+            if reason1:
+                values['reason1'] = _('- Wrong login/password')
+            if reason2:
+                values['reason2'] = _('- User not allowed to have multiple logins')
+            if reason3:
+                values[
+                    'reason3'] = _('- User not allowed to login at this specific time or day')
```

Comment:
please leave this as generic message as explained above.
just let us give the possible reasons...